test/functional/gridfs_stream.test.js

```diff
@@ -133,67 +129,65 @@ describe('GridFS Stream', function () {
    * @example-class GridFSBucket
    * @example-method openUploadStreamWithId
    */
-  it('shouldUploadFromFileStreamWithCustomId', {
+  it('should upload from file stream with custom id', {
     metadata: { requires: { topology: ['single'] } },
 
-    test: function (done) {
-      var configuration = this.configuration;
-      var client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
-      // LINE var MongoClient = require('mongodb').MongoClient,
+    test(done) {
+      const configuration = this.configuration;
+      const client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
+      // LINE const MongoClient = require('mongodb').MongoClient,
       // LINE   test = require('assert');
       // LINE const client = new MongoClient('mongodb://localhost:27017/test');
       // LINE const db = client.db('test);
       // REPLACE configuration.writeConcernMax() WITH {w:1}
       // REMOVE-LINE restartAndDone
       // REMOVE-LINE done();
-      // REMOVE-LINE var db = client.db(configuration.db);
+      // REMOVE-LINE const db = client.db(configuration.db);
       // BEGIN
       client.connect(function (err, client) {
-        var db = client.db(configuration.db);
+        const db = client.db(configuration.db);
         db.dropDatabase(function (error) {
           expect(error).to.not.exist;
 
-          var bucket = new GridFSBucket(db);
-          var readStream = fs.createReadStream('./LICENSE.md');
+          const bucket = new GridFSBucket(db);
+          const readStream = fs.createReadStream('./LICENSE.md');
 
-          var uploadStream = bucket.openUploadStreamWithId(1, 'test.dat');
+          const uploadStream = bucket.openUploadStreamWithId(1, 'test.dat');
 
-          var license = fs.readFileSync('./LICENSE.md');
-          var id = uploadStream.id;
-          test.equal(1, id);
+          const license = fs.readFileSync('./LICENSE.md');
+          const id = uploadStream.id;
+          expect(1).to.equal(id);
```

Comment:
same comment on the commented lines, but also, if we're touching this, we may as well properly assert on actual vs expected (instead of expected vs actual) `expect(id).to.equal(1)`